_posts/2021-05-21-dask-distributed-user-journey.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: The evolution of a Dask Distributed user
+tagline: The critical path to scaling from one machine to thousands
+author: Jacob Tomlinson (NVIDIA)
+tags: [Distributed, Tools, Organisations]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+This week was the 2021 Dask Summit and [one of the workshops](https://summit.dask.org/schedule/presentation/20/deploying-dask/) that we ran covered many deployment options for Dask Distributed.
```

Comment:
It'd be great to also add a link to the workshop video recording (I know it's not up on youtube yet, but will be soon).